src/data/faq_de.json

```diff
@@ -93,8 +93,8 @@
                             "Das Sicherheitszertifikat <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> ist möglicherweise auf Ihrem Gerät deaktiviert. Dies ist eine bekannte Ursache, um folgende Benachrichtigung auszulösen:",
                             "<b>'URSACHE: 2001 Etwas ist schiefgelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",
                             "Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",
-                            "<ul><li>Einstellungen > Biometrische Daten und Sicherheit > Andere Sicherheitseinstellungen > Sicherheitszertifikate anzeigen, SYSTEM</li><li>Scrollen Sie nach unten zu <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Aktivieren Sie das Zertifikat mit dem Schieberegler</li></ul>",
-                            "Sollten diese Schritte das Problem nicht beheben, schreiben Sie bitte Ihren Bericht hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
+                            "<ol><li>'Einstellungen' -> 'Biometrische Daten und Sicherheit' -> 'Andere Sicherheitseinstellungen' -> 'Sicherheitszertifikate anzeigen' -> 'SYSTEM'</li><li>Scrollen Sie nach unten zu <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Aktivieren Sie das Zertifikat mit dem Schieberegler</li></ol>",
+                            "Sollten diese Schritte das Problem nicht beheben, schreiben Sie bitte einen Kommentar hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."

```

Comment:
https://github.com/corona-warn-app/cwa-app-android/issues/968 is already closed.

Reply:
see above